--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -8920,6 +8920,7 @@ static void rtw_cfg80211_init_vht_capab(_adapter *padapter

 static void rtw_cfg80211_create_vht_cap(struct ieee80211_sta_vht_cap *vht_cap)
 {
+#if 0
        u16 mcs_map;
        int i;

@@ -8933,6 +8934,7 @@ static void rtw_cfg80211_create_vht_cap(struct ieee80211_sta_vht_cap *vht_cap)

        vht_cap->vht_mcs.rx_mcs_map = cpu_to_le16(mcs_map);
        vht_cap->vht_mcs.tx_mcs_map = cpu_to_le16(mcs_map);
+#endif
 }

 void rtw_cfg80211_init_wdev_data(_adapter *padapter)
@@ -8961,7 +8963,7 @@ void rtw_cfg80211_init_wiphy(_adapter *padapter)
                if (band) {
                        #if defined(CONFIG_80211N_HT)
                        rtw_cfg80211_init_ht_capab(padapter, &band->ht_cap, BAND_ON_2_4G, rf_type);
-                       rtw_cfg80211_create_vht_cap(&band->vht_cap);
+                       //rtw_cfg80211_create_vht_cap(&band->vht_cap);
                        #endif
                }
        }

